src/app.py

```diff
@@ -17,6 +18,7 @@ def main():
         'results': ResultsEngine(state)
     }
     app = QApplication(sys.argv)
+    plt.rcParams.update({"font.size": 5})
```

Comment:
I think that 5 is too small. Maybe 7?
But it cause problem with plot in results, because text doesn't fit to the window. The ideal solution is to increase font and allow to show text outside the boundaries of window.